packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/SelectFieldDisplayContent.tsx

```diff
@@ -0,0 +1,13 @@
+import { Tag, ThemeColor } from 'twenty-ui';
+
+type SelectFieldDisplayContentProps = {
+  color: ThemeColor;
+  label: string;
+};
+
+export const SelectFieldDisplayContent = ({
+  color,
+  label,
+}: SelectFieldDisplayContentProps) => {
+  return <Tag preventShrink color={color} text={label} />;
+};
```

Comment:
think you should move that to `packages/twenty-front/src/modules/ui/field/display/components` and call it `SelectDisplay`